portfolio/src/main/webapp/js/maps.js

```diff
@@ -0,0 +1,6 @@
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
```

Comment:
what are these coordinates? either add a comment or define a constant to indicate what these mean.